src/textCampaign/services/rumbleUp.service.ts

```diff
@@ -48,6 +61,49 @@ export class RumbleUpService {
     }
   }
 
+  /** starts the compliance registration process */
+  submitComplianceForm(campaign: Campaign, body: ComplianceFormSchema) {
+    //TODO: this should be replaced with an API call to rumble up when its ready to integrate
+    return this.email.sendEmail({
+      to: TCR_COMPLIANCE_EMAIL,
+      subject: 'TCR Compliance Form Submission',
+      message: `
+        <h2>Compliance Information for ${campaign.slug}</h2>
+        <table style="border-collapse: collapse; width: 100%;">
+          <tr>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><strong>EIN:</strong></td>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;">${body.ein}</td>
+          </tr>
+          <tr>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><strong>Name:</strong></td>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;">${body.name}</td>
+          </tr>
+          <tr>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><strong>Address:</strong></td>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;">${body.address}</td>
+          </tr>
+          <tr>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><strong>Website:</strong></td>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><a href="${body.website}">${body.website}</a></td>
+          </tr>
+          <tr>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><strong>Email:</strong></td>
+            <td style="padding: 8px; border-bottom: 1px solid #ddd;"><a href="mailto:${body.email}">${body.email}</a></td>
+          </tr>
+        </table>
+      `,
+    })
+  }
+
+  /** submit the pin code to verify the campaign's compliance email */
+  async submitCompliancePin(pin: string) {
+    // TODO how to we send the pin to rumble up?
```

Comment:
I'm assuming another email 😞 

Based on our conversation after SU this morning w/ Tevyn, even the email above might just be a slack message to the politics team for the time being 😬